src/System Application/App/Extension Management/src/ExtensionSettings.Page.al

```diff
@@ -117,6 +117,26 @@ page 2511 "Extension Settings"
                     ToolTip = 'Specifies the commit ID of the source code for the current version of the project.';
                 }
             }
+
+            group("Build app metadata")
+            {
+                Caption = 'Build app metadata';
+                Visible = IsBuildInformationAvailable;
+
+                field(BuildBy; BuildBy)
+                {
+                    Caption = 'Build by';
```

Comment:
I know this matches the name of the table field, but it's "_built_ by" XYZ.

Reply:
This is surfacing the build properties in the app.json, that's why it's "buld" and not "built":

![image](https://github.com/user-attachments/assets/bdf0dfc5-5e9d-4c14-a32b-3d2cd7a90f35)

Reply:
Added to PR description to avoid confusion.